src/api/index.ts

```diff
@@ -239,19 +239,26 @@ router.post(
 	elapsedTimeMetrics,
 	async (request: Request, res: Response): Promise<void> => {
 
-		const queueName = request.body.queue;
+		const queueName = request.body.queue;   // "installation", "push", "metrics", or "discovery"
+		const jobTypes = request.body.jobTypes || []; // any combination of ["active", "delayed", "waiting", "paused"]
+
+		if(!jobTypes.length){
+			res.status(400);
+			res.send("please specify the jobTypes field (available job types: [\"active\", \"delayed\", \"waiting\", \"paused\"])");
+			return;
+		}
```

Comment:
need this so we can reschedule only the `active` jobs in the installation queue to "free it up" once